amida-tech/amida-messaging-microservice
# Feature/specify users

If this PR fixes a bug, you _must_ add test cases representative of the bug.
#### What's this PR do?
Add the ability to add logs for a user similar to thread. Logs for a user are separate from the messages queue, and need to be containerized separately in the app. Logs may also belong to users that do not have their own authenticated account on Orange-API. These users will have automatically generated emails that end in `@noreply.amida.com`

Changes end points for threads. This PR adds two new columns. A migration script is run automatically, generating the necessary columns to the existing database, and will populate necessary data. This PR also adds support to view participants in a thread.
#### Related JIRA tickets:

- https://jira.amida-tech.com/browse/ORANGE-397
- https://jira.amida-tech.com/browse/ORANGE-408
- https://jira.amida-tech.com/browse/ORANGE-390
- https://jira.amida-tech.com/browse/ORANGE-423

#### How should this be manually tested?
- Run `npm install`
- Using Orange app, create new thread and a new userlog
- Verify that the unread indicators are displaying appropriately, and the correct users are showing as "read"
